app/etc/registration_globlist.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Glob patterns relative to this files parent directory, used by
+ * registration.php to generate a list of includes.
+ */
+return [
+    'code/*/*/cli_commands.php',
```

Comment:
I'd suggest to use root of Magento as a base path (so the paths would start from 'app/code' instead of 'code').
The goal of such change is to make more intuitive for a future reader. I would assume that the reader, when opens the file, would guess from 2 options:
1. Either the files are relative to current folder (`app/etc`)
2. or the files are relative to the root folder

It's difficult to guess that they are relative to `app` because there is no context of `app` here.